build-config/arch/armv8a/busybox.config

```diff
@@ -0,0 +1,1069 @@
+#
+# Automatically generated make config: don't edit
+# Busybox version: 1.24.2
+# Sun May 15 21:25:58 2016
```

Comment:
I'm not grokking why we want an architecture specific busybox.config file.  ONIE should behave the same on all architectures.  This will be difficult to maintain.  Again it may make sense once I read through all the patches.